pkg/restclient/client.go

```diff
@@ -0,0 +1,124 @@
+package restclient
+
+import (
+	"github.com/gluster/glusterd2/pkg/api"
+	bitrotapi "github.com/gluster/glusterd2/plugins/bitrot/api"
+	deviceapi "github.com/gluster/glusterd2/plugins/device/api"
+	eventsapi "github.com/gluster/glusterd2/plugins/events/api"
+	georepapi "github.com/gluster/glusterd2/plugins/georeplication/api"
+	glustershdapi "github.com/gluster/glusterd2/plugins/glustershd/api"
+)
+
+// GlusterD2APIClient contains API client methods for GlusterD2
+type GlusterD2APIClient interface {
+	BitrotAPIClient
```

Comment:
can we rename the interfaces?  these interfaces are just API functions there are not API clients